keps/sig-node/4216-image-pull-per-runtime-class/kep.yaml

```diff
@@ -0,0 +1,47 @@
+title: Image pull per runtime class
+kep-number: 4216
+authors:
+  - "@kiashok"
+owning-sig: sig-node
+participating-sigs:
+  - sig-windows
+  - Kubelet garbage collection WG
+status: implementable
+creation-date: 2023-09-15
+reviewers:
+  - "@jsturtevant"
+  - "@mikebrow"
+  - "@marosset"
+  - "@SergeyKanzhelev"
+  - "@aravindhp"
+  - "@haircommander"
+approvers:
+  - TBD
+
+see-also: []
+replaces: []
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.29"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.29"
+  beta: "v1.31"
+  stable: "v1.35"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: RuntimeClassInCRICalls
```

Comment:
I suggest making this more specific.

Reply:
I changed the feature gate name to 'RuntimeClassInImageCriApi' in the yaml and readme. Let me know if this name sounds better. Thanks